R/opts.R

```diff
@@ -638,12 +638,13 @@ gp_opts <- function(basis_prop = 0.2,
 #' model. Custom settings can be supplied which override the defaults.
 #' @param family Character string defining the observation model. Options are
 #'   Negative binomial ("negbin"), the default, and Poisson.
-#' @param phi A `<dist_spec>` specifying a prior on the overdispersion parameter
-#'   of the reporting process, used only if `familiy` is "negbin". Internally
-#'   parameterised such that the overdispersion is one over the square of this
-#'   prior overdispersion phi. Defaults to a half-normal distribution with mean
-#'   of 0 and standard deviation of 0.25: `Normal(mean = 0, sd = 0.25)`. A lower
-#'   limit of zero will be enforced automatically.
+#' @param dispersion A `<dist_spec>` specifying a prior on the dispersion
+#'   parameter of the reporting process, used only if `familiy` is "negbin".
+#'   Internally parameterised such that the dispersion is one over the square
```

Comment:
I think that phi is still the overdispersion and perhaps that is good to still say